Rubberduck.RibbonDispatcher/AbstractRibbonDispatcher/IRibbonCommon.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+using stdole;
+using Microsoft.Office.Core;
+using System.Runtime.InteropServices;
+
+namespace RubberDuck.RibbonDispatcher {
+    using ControlSize         = RibbonControlSize;
+    using ChangedEventHandler = EventHandler<ChangedControlEventArgs>;
+
+    [ComVisible(true)][CLSCompliant(true)]
+    public interface IRibbonCommon {
```

Comment:
IRibbonControl is already used by MS - though in a different namespace. I selected the best (I could think of) alternative name to avoid confusion. I may still be thinking of VBA terms, where the original implementation was built. Maybe in C# it doesn't matter, with proper namespaces.